packages/checkout/widgets-lib/src/components/Hero/WalletWarningHero.tsx

```diff
@@ -0,0 +1,176 @@
+/* eslint-disable max-len */
+import { Box } from '@biom3/react';
+
+export function WalletWarningHero() {
+  return (
+    <Box
+      testId="wallet-warning-hero"
+      sx={{
+        display: 'flex',
+        justifyContent: 'center',
+        padding: 'base.spacing.x4',
+      }}
+    >
+      <Box>
```

Comment:
Just curious if you need this Box wrapping the svg? Is it because of the css `display`?

Reply:
oopsie, we don't need it